ukis_pysat/data.py

```diff
@@ -74,48 +81,71 @@ def __init__(self, datahub, catalog=None):
     def __enter__(self):
         return self
 
-    def init_catalog(self, catalog=None):
-        """ Initializes PySTAC Catalog
+    def _init_catalog(self):
+        """Initializes an empty Spatio Temporal Asset Catalog."""
+        return pystac.catalog.Catalog(
+            id=str(uuid.uuid4()),
+            description=f"Creation Date: {datetime.datetime.now()}, Datahub: {self.src.value}",
+            catalog_type=pystac.catalog.CatalogType.SELF_CONTAINED,
+        )
 
-        :param catalog: PySTAC Catalog or Collection if already exists
+    def add_items_from_directory(self, item_dir, item_substr=None):

```

Comment:
Please call it `STAC Catalog or STAC Collection` instead of `STAC source` - same for item, add `STAC`.